.github/workflows/ci.yml

```diff
@@ -29,14 +29,38 @@ jobs:
           python-version: "3.8"
 
       - name: Install current package as editable
-        run: pip install -e .
+        run: |
+          pip install -U pip
+          pip install -e .
 
       - name: Install mlstacks package
         run: pip install mlstacks
 
       - name: Check for broken dependencies
         run: pip check
 
+  db-migration-testing:
```

Comment:
I'm guessing this should not run on every PR, but maybe only in the release flow?